clayui.com/src/templates/docs.js

```diff
@@ -153,26 +154,39 @@ export default (props) => {
 		location,
 		pageContext: {blacklist = []},
 	} = props;
+
 	const {
 		allMarkdownRemark,
 		allMdx,
 		markdownRemark,
-		mdTab,
+		mdTabs,
 		mdx,
-		mdxTab,
+		mdxTabs,
 	} = data;
 
 	const {body, excerpt, fields, frontmatter, html, timeToRead} =
 		mdx || markdownRemark;
 
-	const tab = mdTab || mdxTab || {};
+	const tab = mdTabs?.edges[0]?.node || mdxTabs?.edges[0]?.node || {};
+
+	const tabs = [];
+
+	if (mdTabs?.edges.length) {
+		tabs.push(mdTabs.edges[0]?.node);
+	}
+
+	if (mdxTabs?.edges.length) {
+		tabs.push(mdxTabs.edges[0]?.node);
+	}
```

Comment:
I thought it could be done this way:

```
const tabs = mdTabs
	.concat(mdxTabs)
	.map(({node: {frontmatter: {tabName}, html, body}}) => ({tabName, html: html ?? body}));
```

and below we can use `map` to render the tabs.

Reply:
I thought it could be too, but there are a couple of reasons why I didn't go in this direction:
1. At most we have 3 tabs, which is good enough for a `map` but the problem is that the 1st tab, `Overview`,  isn't in the tab list that we get from GraphQL, meaning it has to be manually added before the `map`.
2. The `TabContent` low-level component accepts either `html` or `jsx` prop (it can accept both, but with a caveat), which is a problem: the `Usages` tab works fine, but the `API` tab prioritizes the `html` prop over the `jsx` one, meaning it dangerously sets HTML, as per the instructions within `TabContent`

P.S. I've tested this now, to make sure I'm not going from memory

Reply:
Hmm yeah, makes sense. No problem, so we can continue with this and we can review it later if necessary